lib/matplotlib/backends/backend_wx.py

```diff
@@ -1713,14 +1713,20 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
         rubberBandColor = '#C0C0FF'  # or load from config?
 
         # Set a pen for the border
-        color = wx.NamedColour(rubberBandColor)
+        if wxc.is_phoenix:
+            color = wx.Colour(rubberBandColor)
```

Comment:
NamedColour is defined in wx_compat, so you could just use wxc.NamedColour(rubberBandColor)